### PR TITLE
Select openOCD board command

### PR DESCRIPTION
**Pull request by brianignacio5**
Add command `Select OpenOCD Board Configuration` to choose any of the boards defined in OpenOCD-esp32 Scripts's esp-config.json

### CODE REVIEW

**Comment by pwmb** [comment]
@brianignacio5 PTAL 634d146

File: src/extension.ts [hunk 1]
```diff
@@ -1197,6 +1198,48 @@ export async function activate(context: vscode.ExtensionContext) {
...
+        };
+      });
+      const selectOpenOCdConfigsMsg = locDic.localize(
+        "extension.enterOpenOcdConfigMessage",
+        "Enter OpenOCD Configuration File Paths list"
+      );
+      const selectedBoard = await vscode.window.showQuickPick(choices, {
+        placeHolder: selectOpenOCdConfigsMsg,
+      });
+      if (!selectedBoard) {
```
Comment (pwmb) [inline src/extension.ts:1218]: When User select a board, I think we can show a info message that board selection is successful

**Review by pwmb — APPROVED** [review]
LGTM, thanks for the PR

Left one suggestion

**Comment by brianignacio5** [comment]
> @brianignacio5 PTAL [634d146](https://github.com/espressif/vscode-esp-idf-extension/commit/634d14601b7caed7a5073418722d5e06cf215fb4)

LGTM !

**Review by pwmb — APPROVED** [review]
LGTM

**Pull request merged**